src/GojsDiagram.test.tsx

```diff
@@ -0,0 +1,150 @@
+import * as React from 'react';
+import * as go from 'gojs';
+import { Diagram } from 'gojs';
+import { mount } from 'enzyme';
+import GojsDiagram from './GojsDiagram';
+
+describe('GojsDiagram component', () => {
+
+    const model = {
+        nodeDataArray: [
+            { key: 'Alpha', color: 'lightblue' },
+            { key: 'Beta', color: 'orange' },
+            { key: 'Gamma', color: 'lightgreen' },
+            { key: 'Delta', color: 'pink' },
+            { key: 'Omega', color: 'grey' }
+        ],
+        linkDataArray:
+            [
+                { from: 'Alpha', to: 'Beta' },
+                { from: 'Alpha', to: 'Gamma' },
+                { from: 'Beta', to: 'Delta' },
+                { from: 'Gamma', to: 'Omega' }
+            ]
+    };
+
+    const createDiagram = (diagramId: string): Diagram => {
+        const $ = go.GraphObject.make;
+
+        const myDiagram: Diagram = $(
+            go.Diagram,
+            diagramId,
+            {
+                initialContentAlignment: go.Spot.LeftCenter,
+            });
+
+        myDiagram.nodeTemplate =
+            $(
+                go.Node,
+                'Auto',
+                $(
+                    go.Shape,
+                    'RoundedRectangle',
+                    { strokeWidth: 0 },
+                    new go.Binding('fill', 'color')),
+                $(
+                    go.TextBlock,
+                    { margin: 8 },
+                    new go.Binding('text', 'key'))
+            );
+
+        return myDiagram;
+    };
+
+    const myDiagramId = 'myDiagramId';
+
+    let diagram: Diagram;
+    let wrapper;
+
+    beforeEach(() => {
+        const dom = document.body;
+        wrapper = mount(
+            (
+                <GojsDiagram
+                    diagramId={myDiagramId}
+                    model={model}
+                    createDiagram={(id) => {
+                        diagram = createDiagram(myDiagramId);
+                        return diagram;
+                    }
+                    }
+                    className="fakecss"
+                />
+            ),
+            { attachTo: dom });
+    });
+
+    it('renders the model provided as parameter', () => {
+        checkRenderedModel(model);
+        wrapper.unmount();
+    });
+
+    const checkRenderedModel = (currentModel) => {
```

Comment:
change to  `checkIfModelDiagramRendersModel(diagram,  model)`  to make it more readable?

Reply:
Or `diagramRendersModel(diagram, model)`